{"name": "Авторизация курьера — проверка ошибок", "status": "passed", "description": "Проверка авторизации негативными сценариями", "steps": [{"name": "Сценарий: Неправильный пароль", "status": "passed", "start": 1757418723037, "stop": 1757418723667}, {"name": "Проверить код ответа", "status": "passed", "start": 1757418723667, "stop": 1757418723667}, {"name": "Проверить сообщение об ошибке", "status": "passed", "start": 1757418723667, "stop": 1757418723667}], "parameters": [{"name": "description", "value": "'Неправильный пароль'"}, {"name": "creds", "value": "{'login': 'login_user_456', 'password': 'wrong_pass'}"}, {"name": "expected_status", "value": "404"}, {"name": "expected_message", "value": "'Учетная запись не найдена'"}], "start": 1757418723037, "stop": 1757418723667, "uuid": "2a05923b-9ec3-4750-be8d-31f0569541e8", "historyId": "d40d110f8a6c5b6a7fb7090dde9e5280", "testCaseId": "19c43b3cacae3758579e377968a7e303", "fullName": "tests.test_login_courier.TestCourierLogin#test_login_errors", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "Ksu"}, {"name": "thread", "value": "18628-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}], "titlePath": ["tests", "test_login_courier.py", "TestCourierLogin"]}